tests/unit/plugins/modules/web_infrastructure/test_jenkins_build.py

```diff
@@ -93,6 +106,39 @@ def test_module_create_build(self, jenkins_connection, test_deps):
             })
             jenkins_build.main()
 
+    @patch('ansible_collections.community.general.plugins.modules.web_infrastructure.jenkins_build.test_dependencies')
+    @patch('ansible_collections.community.general.plugins.modules.web_infrastructure.jenkins_build.JenkinsBuild.get_jenkins_connection')
+    def test_module_stop_build(self, jenkins_connection, test_deps):
+        test_deps.return_value = None
+        jenkins_connection.return_value = JenkinsMock()
+
+        with self.assertRaises(AnsibleExitJson):
+            set_module_args({
+                "name": "host-check",
+                "build_number": "1234",
+                "state": "stopped",
+                "user": "abc",
+                "token": "xyz"
+            })
+            jenkins_build.main()
+
+    @patch('ansible_collections.community.general.plugins.modules.web_infrastructure.jenkins_build.test_dependencies')
+    @patch('ansible_collections.community.general.plugins.modules.web_infrastructure.jenkins_build.JenkinsBuild.get_jenkins_connection')
+    def test_module_stop_build_idempotent(self, jenkins_connection, test_deps):
+        # The stopped_build function is idempotent.
+        test_deps.return_value = None
+        jenkins_connection.return_value = JenkinsMock()
+
+        with self.assertRaises(AnsibleExitJson):
+            set_module_args({
+                "name": "host-check",
+                "build_number": "1234",
+                "state": "stopped",
+                "user": "abc",
+                "token": "xyz"
+            })
+            jenkins_build.main()
```

Comment:
For proper idempotence, the module needs to return `changed=false` since the build has already been stopped.

Reply:
As I am not that familiar with Ansible Module enough, could you tell me how to validate changed=false in this test case? Thank you.